Gemfile.lock

```diff
@@ -108,3 +108,6 @@ DEPENDENCIES
 
 RUBY VERSION
    ruby 2.6.4p104
+
+BUNDLED WITH
```

Comment:
Im not sure if this is required or not

Reply:
@Copinmalin would you mind removing this? I prefer not to introduce any potential issues as long as you can build locally without it, etc.

Reply:
Done